index.js

```diff
@@ -478,7 +478,7 @@ class GithubScm extends Scm {
 
     /**
      * Get a list of names and references of opened PRs
-     * @async  _getOpenedPRs
+     * @async  getOpenedPRs
```

Comment:
Hmm, `_getOpenedPRs` and `_ getChangedFiles ` are the function names

Reply:
it's not really consistent in the file 🤷‍♀